cpp/daal/include/services/internal/sycl/math/mkl_lapack.h

```diff
@@ -55,7 +55,9 @@ struct MKLPotrf
     Status operator()(const math::UpLo uplo, const size_t n, Buffer<algorithmFPType> & a, const size_t lda,
                       cl::sycl::buffer<algorithmFPType, 1> & scratchpad)
     {
+
```

Comment:
Why mkl_lapack.h is changed?

Reply:
Sorry, I've tried to figure out where constants are converted into int64_t